crypto/src/vrf/schnorrkel/mod.rs

```diff
@@ -0,0 +1,158 @@
+use crate::random::{CryptoRng, Rng};
+use parity_scale_codec::{Decode, Encode};
```

Comment:
Is there an issue depending on `serialization` rather than `parity_scale_codec`? (Few other places below too)